### PR TITLE
Add authorship order

### LINKED ISSUE #105: Determine Author Order
Right now, the comment says: "#Below is a randomised order of authors. The author order will be decided through discussion and mutual consent." This is a note so that we remember :)

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose

closes #105
### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -18,6 +18,14 @@ authors:  #Below is a randomised order of authors. The author order will be deci
     funders:
       - Neurofibromatosis Therapeutic Acceleration Program
       - Children's Tumor Foundation
+  -
+    github: jaclyn-taroni
+    name: Jaclyn N Taroni [1]
```
Comment (jaclyn-taroni) [inline content/metadata.yaml:28]: I do not feel that my contributions were equal to those of @jaybee84! I am happy to go with the proposed ordering (e.g., dropping `[1]`) if that's a-okay with everyone else.

Comment (jaybee84) [reply content/metadata.yaml:28]: I would like to re-propose equal authorship for you @jaclyn-taroni :)

Comment (jaclyn-taroni) [reply content/metadata.yaml:28]: Okay, I will not die on this hill 😅

Comment (jaybee84) [reply content/metadata.yaml:28]: :)

**Comment by jaybee84** [comment]
@jaclyn-taroni : since you already reviewed the PR, would you mind approving it ?

**Review by jaclyn-taroni — APPROVED** [review]
👍

**Pull request merged**